scripts/parse3.py

```diff
@@ -451,7 +451,7 @@ def ParseData(data, start, end, messages, depth = 0):
             #start = start + 9
             start = start + 8
             try:
-                floatNum = struct.unpack('d',struct.pack('q',hex(num),16))

```

Comment:
What is the reasoning behind this? I wonder if we need to port the changes to ILEAPP as well since the same file is being used there afaik. Is this not a dependency we can `pip install` so both projects can benefit from bug fixes? I haven't heard of this package before so no idea.

Reply:
You can't use hex(num) as that returns a string. This is a bug in the library.

Reply:
That's a bug in the library. hex(num) returns a string, but it needs a number there.

Reply:
yep, `hex()` does return a `str`. makes sense 👍